jbmc/src/jbmc/jbmc_parse_options.cpp

```diff
@@ -1022,7 +1022,14 @@ void jbmc_parse_optionst::help()
     "Usage:                       Purpose:\n"
     "\n"
     " jbmc [-?] [-h] [--help]      show help\n"
-    " jbmc class                   name of class to be checked\n"
+    " jbmc class                   name of class or JAR file to be checked\n"
+    "                              In the case of a JAR file, if a main class"
+    " can be inferred\n"
+    "                              from --main-class, --function or the JAR"
+    " manifest (checked in\n"
+    "                              this order), the behavior is the same as"
+    " running jbmc on the\n"
```

Comment:
:pick: line wrapping is weird

Reply:
Fixed (I think - it's similar to what was done below in the same file).